src/WebJobs.Script.Analyzers/WebJobsAttributeAnalyzer.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Immutable;
+
+namespace Microsoft.Azure.Functions.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class WebJobsAttributeAnalyzer : DiagnosticAnalyzer
+    {
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get { return ImmutableArray.Create(DiagnosticDescriptors.IllegalFunctionName); } }
+
+        public override void Initialize(AnalysisContext context)
+        {
+            // https://stackoverflow.com/questions/62638455/analyzer-with-code-fix-project-template-is-broken
```

Comment:
I would prefer to; it might save someone time/frustration in the future. (When I picked up analyzer work again after a break, I forgot to add the line below and lost a chunk of time figuring out why nothing was working. I have every reason to believe this will happen again. Probably to me.)